src/Microsoft.IdentityModel.Tokens/Json/JsonSerializerPrimitives.cs

```diff
@@ -695,6 +706,122 @@ internal static string ReadStringOrNumberAsString(ref Utf8JsonReader reader, str
             return retVal;
         }
 
+        internal const string TryToCreateDateTimeClaimsSwitch = "Switch.Microsoft.IdentityModel.TryAllStringClaimsAsDateTime";
```

Comment:
Do we really need a switch for this? Or is this more of a safety measure to disable quickly if there's some unintended behaivor?

Reply:
exactly. It is unlikely, but we wanted an opt out.